release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -0,0 +1,863 @@
+install it if you plan on using any of the services from the OpenShift Ansible
+broker or template service broker. In {product-title} 4.1, the Service Catalog
+API server is installed into the `openshift-service-catalog-apiserver` namespace
+and the Service Catalog controller manager is installed into the
+`openshift-service-catalog-controller-manager` namespace. In {product-title}
+3.11, both of these components were installed into the `kube-service-catalog`
+namespace.
+
+[discrete]
+[[ocp-4-1-template-service-broker-no-longer-installed-by-default]]
+==== Template Service Broker no longer installed by default
+
+The Template Service Broker is not installed by default in {product-title} 4.1.
+Cluster administrators can install the Template Service Broker if users will
+need access to template applications from the web console.
+
+[discrete]
+[[ocp-4-1-oc-adm-commands-deprecated]]
+==== Several `oc adm` commands are now deprecated
+
+Deprecated `oc adm` commands include:
+
+* `oc adm create-master-certs` - Create keys and certificates
+* `oc adm create-key-pair` - Create an RSA key pair.
+* `oc adm create-server-cert` - Create a key and server certificate.
+* `oc adm create-signer-cert` - Create a self-signed CA.
+
+[discrete]
+[[ocp-4-1-configurability-of-imagepolicyadmission]]
+==== The configurability of the imagepolicyadmission plug-in is not present
+
+The configurability of the `imagepolicyadmission` plug-in is not present in
+{product-title} 4.1. The plug-in runs, but currently only with default
+configuration. Configuring it requires using the unsupported overrides
+mechanism.
+
+[[ocp-4-1-technology-preview]]
+== Technology Preview features and feature gates
+
+Some features in this release are currently in Technology Preview. These
+experimental features are not intended for production use. Please note the
+following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
+Features Support Scope]
+
+Starting in {product-title} 4.1, some _feature gates_, a set of key=value pairs
+that describe Technology Preview features, will ship enabled. You can change the
+default feature gates, but doing so prevents you from upgrading your cluster.
+Any change to a feature gate stops an upgrade. Clusters with a changed feature
+gate must be re-installed to apply even an asynchronous release update.
+
+In the table below, features marked *TP* indicate _Technology Preview_ and
+features marked *GA* indicate _General Availability_.
+
+If a feature is marked *yes;on*, that feature is controlled by a feature gate
+that is enabled, or turned on, when shipped. If a feature is marked *yes;off*,
+that feature is controlled by a feature gate that is disabled, or turned off,
+when shipped.
+
+.Technology Preview Tracker
+[cols="4",options="header"]
+|====
+|Feature |Feature gate |OCP 3.11 |OCP 4.1
```

Comment:
The "Feature gate" column is empty.

Reply:
@liangxia Yes, to my knowledge, nothing officially has a feature gate yet, but that can change in a future async release. 

@joanne-okerman Is this still the case? I used the Tech Preview tracker spreadsheet as my source of truth for this information.